docker-compose.yaml

```diff
@@ -0,0 +1,29 @@
+version: '3.8'
```

Comment:
i am not sure about best practice here. can we somehow differentiate that this is a docker image only to be used inside php-cs-fixer itself, and not the docker image for ppl willing to run PHP CS Fixer?

maybe we can extract it to separated package, like https://github.com/PHP-CS-Fixer/php-docker-toolkit (whatever name) ?

Reply:
I don't think `docker-compose.yaml` are meant to be shared between different projects. I'd say we can safely assume this will be considered as the setup to work on the project itself, not to use it elsewhere.